### DISCUSSION TITLE
Can I repeat `LEADER` after some action is triggered

### DISCUSSION THREAD

**Discussion by ho-oto** [wezterm/wezterm#2034] [Q&A] [answered]
I'm trying to reproduce my `.tmux.conf` in wezterm by treating `LEADER` as a prefix of tmux.
In tmux, we can run multiple commands with a single press of prefix by adding `-r` (repeat-time) option like
```
bind -r S-left resize-pane -L 5
```

Can I do similar thing in wezterm.

**Comment by MuhammedZakir** [comment, marked as answer]
`LEADER` itself doesn't have that option. But you can do that using `key tables`.

1. https://wezfurlong.org/wezterm/config/key-tables.html
2. See the `one_shot` option: https://wezfurlong.org/wezterm/config/lua/keyassignment/ActivateKeyTable.html
